netkan/Dockerfile

```diff
@@ -3,12 +3,9 @@ RUN useradd -ms /bin/bash netkan
 ADD . /netkan
 WORKDIR /netkan
 RUN pip install pip --upgrade
-RUN pip install mypy
 RUN chown -R netkan:netkan /netkan
 USER netkan
 RUN pip install --user . --no-warn-script-location --force-reinstall
-RUN mypy .
-RUN python -m unittest -v
 
```

Comment:
Our docker environment isn't our github one, we likely still want to run our tests inside the container.

Reply:
Makes sense to me.